source/mainnet/net/installation/downloads.rst

```diff
@@ -59,33 +87,63 @@ The |bw| is available to download as an extension for the `chromium web browsers
 Concordium Desktop Wallet
 =================================
 
-Windows v1.5.0
---------------
+.. _dw-mainnet-download:
+
+.. dropdown:: Mainnet
+
+   .. dropdown:: Windows v1.5.0
```

Comment:
These version numbers kind of reads like it is Windows that is in version 1.5.0. Same for Linux and MacOS. The other wallets doesn't show version number in the download link, so it might not be needed?

Reply:
I was asked to include version numbers a while ago. These have been there for a while, also on the currently released site.